src/menu.rb

```diff
@@ -0,0 +1,175 @@
+require './book'
+require './rental'
+require './student'
+require './teacher'
+require './persons'
+
+class App
+  def initialize
+    @book_list = []
+    @people_list = []
+    @rental_list = []
+    @user_menu = "
+			Please choose an option by entering a number:
+			1 - List all books
+			2 - List all people
+			3 - Create a person
+			4 - Create a book
+			5 - Create a rental
+			6 - List all rentals for a given person id
+			7 - Exit
+		"
+  end
+
+  def menu
+    loop do
+      puts @user_menu
+      @input = gets.chomp.to_i
+      users_selection(@input)
+      break if @input == 7
+    end
+  end
+
+  def users_selection(users_input)
+    case users_input
+    when 1
+      list_books
+    when 2
+      all_people
+    when 3
+      create_person
+    when 4
+      create_book
+    when 5
+      create_rental
+    when 6
+      list_all_rentals_by_id
+    else
```

Comment:
- [ ] I can see you used the menu method to replace the main method suggested in the project requirements. As demanded by project requirements, please consider moving this to your menu method.